crates/bws/src/config.rs

```diff
@@ -58,6 +66,34 @@ fn get_config_path(config_file: Option<&Path>, ensure_folder_exists: bool) -> Pa
     config_file
 }
 
+pub(crate) fn get_state_file_path(
+    state_file: Option<PathBuf>,
+    profile: Option<String>,
+    ensure_folder_exists: bool,
+) -> PathBuf {
+    let state_file = state_file.unwrap_or_else(|| {
+        let base_dirs = BaseDirs::new().unwrap();
+        let state_filename = match profile {
+            Some(p) => p + "-" + STATE_FILENAME,
+            None => "default-".to_string() + STATE_FILENAME,
```

Comment:
You can move the `-` into the suffix.

Reply:
Resolved in [566545d](https://github.com/bitwarden/sdk/pull/388/commits/566545d475abb1fd04eb825090e4227df841f4ed)